CLAUDE.md

```diff
@@ -0,0 +1,141 @@
+# CLAUDE.md - Agent Instructions
+
+## Project Context
+
+mUSD is a decentralized stablecoin protocol built for the Mezo network. It allows Bitcoin holders to mint mUSD stablecoins by creating Collateralized Debt Positions (CDPs) using their BTC as collateral. The protocol maintains a $1 USD peg through arbitrage mechanisms, liquidations, and redemptions.
+
+The system is based on Threshold USD (a Liquity fork) with key enhancements including:
+- Fixed-rate borrowing with refinancing capabilities
+- Protocol Controlled Value (PCV) for fee management
+- EIP-712 signature verification for delegation
+- Upgradeable contracts for flexibility
+
+## Commands
+
+**Primary development happens in the `solidity/` directory.**
+
+- Build: `cd solidity && pnpm build` 
+- Test: `cd solidity && pnpm test`
+- Deploy: `cd solidity && pnpm deploy --network matsnet`
```

Comment:
Is `matsnet` still valid in the musd repo? Haven't we migrated to `testnet` yet?